examples/twodnavierstokes_decaying.jl

```diff
@@ -25,13 +25,13 @@ nothing # hide
 #
 # First, we pick some numerical and physical parameters for our model.
 
-n, L  = 128, 2π             # grid resolution and domain length
+n, L  = 256, 2π             # grid resolution and domain length
 nothing # hide
 
 ## Then we pick the time-stepper parameters
-    dt = 1e-2  # timestep
-nsteps = 4000  # total number of steps
- nsubs = 1000  # number of steps between each plot
+    dt = 5e-3  # timestep
+nsteps = 8000  # total number of steps
```

Comment:
This may be a pretty heavy example for docs. But we'll see how long they take to build; maybe it will be fine.

Reply:
128 would work OK also... but 256 looks better.

The problem will arise when we make **all** examples producing 256^2 animations...